.github/workflows/integration-tests.yml

```diff
@@ -143,6 +143,7 @@ jobs:
       matrix:
         key: [8, 16]
         mode: [normal, random]
+        auth: [None, Basic]
```

Comment:
With this line we will run 8 tests in parallel. But it is enough to run a single test with auth enabled.
Please, use `include` to add one specific configuration: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations

This line will be `auth: [None]` and include should add `(key: 8, mode: normal, auth: Basic)`